app/controllers/bands_controller.rb

```diff
@@ -15,11 +18,43 @@ def create
     end
   end
 
+  def edit
+    @band = Band.find(params[:id])
+  end
+
+  def update
+    @band = Band.find(params[:id])
+
+    if @band.update(band_params)
+      flash[:notification] = "Changes have been made"
+      redirect_to "/bands/show/#{params[:id]}"
```

Comment:
unfortunately, this path looks wrong, which suggests that maybe you didnt test it...?

`/bands/#{params[:id]}` is more what you want

I would also change the message to "Band updated" or "your changes have been made"